run.py

```diff
@@ -553,6 +553,13 @@ def fetch_test_details(var) -> dict:
         details["status"] = "Not Executed"
         return details
 
+    if store and reuse is not None:
+        log.error(
+            "--store and --reuse are mutually exclusive arguments.\nPlease use --store for capturing snapshot"
+            " and --reuse for using the captured snapshot"
+        )
+        sys.exit(1)
+
```

Comment:
Can we reuse `docopts` functionality instead of this check statement? I believe it is possible to mention `<this-option> | <either-this-option>`

Reply:
Implemented using docopts options.